src/styles/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 0 16px;
+  flex-shrink: 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: $white;
+  transition: all 0.3s ease-in-out;
+
+  &__product-name {
+    margin: 40px 0 4px;
+    width: 100%;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__product-name {
+      color: #34568b;
+    }
+  }
+
+  &__product-img {
+    margin: 32px auto 0;
+    width: 160px;
+    height: 134px;
+    display: flex;
+  }
+
+
+  &__product-code {
+    margin-bottom: 16px;
+    line-height: 14px;
+    color: $secondary-accent;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__product-price {
+    margin: 24px 0 16px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &_text {
+      color: $secondary-accent;
+      font-size: 12px;
+    }
+
+    &_value {
+      color: $main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__purchase-link {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+    margin-bottom: 16px;
+    width: 100%;
+    height: 40px;
+    flex-shrink: 0;
+    border: none;
+    border-radius: 5px;
+    background: $blue-accent;
+    color: $white;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+  }
+
+  &__purchase-link:hover {
+    box-sizing: border-box;
+    background-color: $white;
+    color: $blue-accent;
+    cursor: pointer;
+    border: 1px solid $blue-accent;
+    border-radius: 5px;
+  }
```

No need to duplicate selectors, fix it everywhere
```suggestion
  &__purchase-link {
    display: flex;
    justify-content: center;
    text-align: center;
    align-items: center;
    margin-bottom: 16px;
    width: 100%;
    height: 40px;
    flex-shrink: 0;
    border: none;
    border-radius: 5px;
    background: $blue-accent;
    color: $white;
    line-height: 16px;
    font-size: 14px;
    font-weight: 700;
    text-decoration: none;
    
    &:hover {
      // write styles here
    }
  }
```